tests/settings/ConfigurationGatheringTest.py

```diff
@@ -470,9 +470,9 @@ def test_get_filtered_bears(self):
         local_bears['cli'] = _sort_bears(local_bears['cli'])
         global_bears['cli'] = _sort_bears(global_bears['cli'])
 
-        self.assertEqual(len(local_bears['cli']), 2)
-        self.assertEqual(str(local_bears['cli'][0]),
-                         "<class 'JavaTestBear.JavaTestBear'>")
+        self.assertEqual(len(local_bears['cli']), 3)
         self.assertEqual(str(local_bears['cli'][1]),
```

Comment:
Why not assert the instances of all three bears.

Reply:
Good idea, made the change :+1: